pkg/language/language.go

```diff
@@ -10,10 +10,16 @@ import (
 	"github.com/wakatime/wakatime-cli/pkg/log"
 )
 
+// Config defines language detection options.
+type Config struct {
+	// GuessLanguage enables detecting lexer language from file contents
```

Comment:
just ot be consistent with other comments.
```suggestion
	// GuessLanguage enables detecting lexer language from file contents.
```